packages/fetchai/skills/tac_control_contract/behaviours.py

```diff
@@ -171,36 +183,47 @@ def _request_mint_items_transaction(self, game: Game) -> None:
 
         :return: None
         """
-        self.context.logger.info("requesting mint_items transactions.")
-        for agent_state in game.initial_agent_states.values():
```

Comment:
We cannot send them all at once with current nonce strategy..at some point we need to have a more elaborate nonce creation strategy